<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetPlatformTypeDialog</class>
 <widget class="QDialog" name="TargetPlatformTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Target Platform Type</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="3">
    <widget class="QGroupBox" name="mSignaturesGroupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox{border:0}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="4" column="3">
       <widget class="QLabel" name="mAcousticSigUnitLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>db</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="mSigHeaderUnitsLabel">
     <property name="text">
      <string>Units</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="mRadarSigUnitLabel">
     <property name="text">
      <string>m^2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mAcousticSigLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Acoustic Signature</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="mInfraredSigValueEdit"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="mPlatformTypeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="mAcousticSigValueEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="mOpticalSigValueEdit"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="mOpticalSigUnitLabel">
     <property name="text">
      <string>m^2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="mSigHeaderValueLabel">
     <property name="text">
      <string>Constant Value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mRadarSigLabel">
     <property name="text">
      <string>Radar Signature</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mSigHeaderLabel">
     <property name="text">
      <string>Signature Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="mSigHeaderTypeLabel">
     <property name="text">
      <string>TYPE NAME</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mRadarSigEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mInfraredSigEditLabel">
     <property name="text">
      <string>Infrared Signature</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="mInfraredSigUnitLabel">
     <property name="text">
      <string>W/Sr</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mOpticalSigEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="mUserDefinedTypeButton">
     <property name="text">
      <string>User Defined Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mOpticalSigLabel">
     <property name="text">
      <string>Optical Signature</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mInfraredSigEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="mRadarSigValueEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="mAcousticSigEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="mTargetPlatformTypeButton">
     <property name="text">
      <string>TARGET_PLATFORM_TYPE</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TargetPlatformTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TargetPlatformTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
